src/lib/seam/components/SupportedDeviceTable/ShowAllDevicesButton.tsx

```diff
@@ -0,0 +1,32 @@
+import { ChevronRightIcon } from 'lib/icons/ChevronRight.js'
+
+interface ShowAllDevicesButtonProps {
+  onClick: () => void
+  isShowing: boolean
+  viewingAllDevices: boolean
+  totalDeviceCount: number
+}
+
+export function ShowAllDevicesButton({
+  onClick,
+  isShowing,
+  viewingAllDevices,
+  totalDeviceCount,
+}: ShowAllDevicesButtonProps) {
+  if (!isShowing) {
+    return null
+  }
+
+  const label = viewingAllDevices ? t.showLess : t.seeAll(totalDeviceCount)
+
+  return (
+    <button className='show-all-devices-button' onClick={onClick}>
+      <ChevronRightIcon /> {label}
+    </button>
+  )
+}
+
+const t = {
+  showLess: 'Show less',
+  seeAll: (count: number) => `See all ${count} devices`,
```

Comment:
To match show less
```suggestion
  showAll: (count: number) => `See all ${count} devices`,
```

Reply:
Updated in https://github.com/seamapi/react/pull/251/commits/ca147673afef77a36d68d1dc52cb5e3b190d468a